file-formats/ilmb/type/zif_aff_ilmb_v1.intf.abap

```diff
@@ -0,0 +1,108 @@
+INTERFACE zif_aff_ilmb_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Condition Fields Direct</p>
+    "! Condition fields direct
+    BEGIN OF ty_cond_fields_direct,
+      "! <p class="shorttext">Condition Field</p>
+      "! Condition field
+      condition_field TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Source Table</p>
+      "! Source table
+      source_table    TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Source Field</p>
+      "! Source field
+      source_field    TYPE zif_aff_types_v1=>ty_object_name_30,
+    END OF ty_cond_fields_direct,
+    "! <p class="shorttext">Condition Fields Direct</p>
+    "!  Condition fields direct
+    ty_cond_fields_directs TYPE SORTED TABLE OF ty_cond_fields_direct WITH UNIQUE KEY source_table source_field.
+
+  TYPES:
+    "! <p class="shorttext">Available Condition Fields</p>
+    "! Available condition fields
+    BEGIN OF ty_available_condition_field,
+      "! <p class="shorttext">Condition Field</p>
+      "! Condition field
+      condition_field             TYPE zif_aff_types_v1=>ty_object_name_30,
```

Comment:
Usually, we suggest to use `name` to avoid repeatation.

```suggestion
      name             TYPE zif_aff_types_v1=>ty_object_name_30,
```

Reply:
Changed to name.